packages/fiber/src/core/hooks.tsx

```diff
@@ -22,19 +22,34 @@ export type BranchingReturn<T, Parent, Coerced> = ConditionalType<T, Parent, Coe
 type noop = (...args: any[]) => any
 type PickFunction<T extends noop> = (...args: Parameters<T>) => ReturnType<T>
 
+export function useMemoizedFn<T extends noop>(fn?: T): PickFunction<T> {
```

Comment:
what's this being used for? should this be inlined to canvas since that's the only place where it's being used? i assume it's not officially exported? i think this should go to utils.

Reply:
This memoizes onPointerMissed from #2146. I'll move it to utils since it's used privately in canvas.